hack/verify-lint.sh

```diff
@@ -24,6 +24,7 @@ for d in $(find . -type d -a \( -iwholename './pkg*' -o -iwholename './cmd*' \))
 		 --exclude='.*_test\.go:.*error return value not checked.*\(errcheck\)$' \
 		 --exclude='duplicate of.*_test.go.*\(dupl\)$' \
 		 --exclude='.*/mock_.*\.go:.*\(golint\)$' \
+		 --exclude='declaration of "err" shadows declaration at.*\(vetshadow\)' \
```

Comment:
The problem is that the check is known as error prone. It sometimes report error and sometimes not, and I don't think something like:
```
if err := func1(); err != nil {
}
if err := func2(); err != nil {
}
```
should be an error.